Song_sort_Title.css

```diff
@@ -67,62 +41,152 @@ li{
 }
 
 
+
 nav a{
 	display:block;
 	width:100px;
 	padding: 20px 15px;
 	text-decoration:none;
-	font-family:Arial;
-	color:black;
+	font-family: 'Open Sans', sans-serif;
+	font-weight:bold;
+	color:white;
 	text-align: center;
 }
 
 nav a:hover{
-	background:purple;
+	background:black;
 	transition:0.5s;
 }
 
-.content_1{
-	background-color: white;
+ul li ul{
+	padding: 0;
+	position: absolute;
+	top: 58px;
+	width: 100px;
+	display: none;
+	visibility: hidden;
+	z-index: 5;
+}
+	
+ul li ul li{
+	background-color:  #0C59CF;
+	display: block;
+}
+
+ul li ul li:hover{background: black;}
+
+ul li:hover ul{
+	display: block;
+	visibility: visible;
+}
+
+.logo img{
+	position:absolute;
+	margin-top:-20px;
+}
+
+
+
+.img_1{
 	grid-column-start: 1;
 	grid-column-end: span 4;
-	z-index: 2;	
-	height: 100px;
-	border: 2px solid red;
 }
 
+.img_1 img{
+	display:block;
+	margin-left:auto;
+	margin-right:auto;
+	height:400px;
+	width:100%;
+}
+
+.footer{
+	background-color: #0C59CF;
+	grid-column-end: span 4;
+	padding-top:25px;
+	text-align: center;
+	color:white;
+	height:50px;
+	font-family: 'Open Sans', sans-serif;
+	font-weight:300;
+}
+
+
+.content_1{
+	grid-column-start: 2;
+	grid-column-end: span 4;
+	background-color: white;
+	z-index: 2;
+	height:200px;
+	text-align:center;
+}
+
+.section3{
+	background-color: white;
+	width: 85%;
+	height: 200px;
+	position: absolute;
+	z-index: 4;
+}
+
+
+
+
+.section3 h1{
+	font-size:24px;
+	color:black;
+	margin-top:50px;
+	font-family: 'Source Sans Pro', sans-serif;
+	font-weight:600;
+}
 
 .content_2{
 	grid-column-start: 1;
-	grid-column-end: span 1;
+	grid-column-end: 2;
 	background-color: white;
 	z-index: 2;
-	height:604px;
-	border: 2px solid red;
+	height:200px;
+}
+.content_2 img{
+	height:200px;
+	width:202px;
+	
 }
-
 
 .content_3{
-	grid-column-start: 2;
+	grid-column-start: 1;
 	grid-column-end: span 4;
-	background-color:white;
-	z-index: 2;
-	height:600px;		
+	background-color: black;
+	z-index: 1;
+	color:white;
 }
 
 .content_3{
 	width: 100%;
 	display: grid;
     grid-gap: 2%;
-	grid-template-columns: 2% 12% 12% 12% 12% 12% 12%;
-    grid-template-rows: 26px;
-	padding-top: 4px;
-	padding-left:5%;
+	grid-template-columns: 6% 14% 14% 14% 14% 12% 12%;
+    grid-template-rows: 70px;
+	font-family: 'Source Sans Pro', sans-serif;
+	font-weight:600;
+}
+
+.content_4{
+	grid-column-start: 1;
+	grid-column-end: span 4;
+	background-color: white;
+	z-index: 2;
+	height:600px;
+	font-family: 'Open Sans', sans-serif;
+	font-weight:300;
+}
+
+.content_4{
+	width: 100%;
+	display: grid;
+	grid-gap: 2%;
+	grid-template-columns: 6% 14% 14% 14.5% 16% 10.5% 12%;
 	overflow:scroll;	
 	overflow-x:scroll;
 	overflow-x: hidden;	
 }
-
-h1{
```

Comment:
21/9: I delete this h1 since this h1 was use in layout one when I align the text in the side bar. But now I don't apply it then it better if I remove it so it doesn't causes any confusion.